src/kOS/Function/Misc.cs

```diff
@@ -28,6 +28,51 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+[Function("hudtxt")]
+    public class FunctionHudTxt : FunctionBase
+    {
+
+        public override void Execute(SharedObjects shared)
+
+        {
+            int mirror          = Convert.ToInt32(shared.Cpu.PopValue());
+            string color        = shared.Cpu.PopValue().ToString();
```

Comment:
Note, none of these ideas should prevent the Pull Request.  They're just ideas for a further refinement prior to a release if you like.  (They would have to be done prior to release or not at all, though, otherwise we have backward compatibility problems)